src/99-appendices/08-coordinate-systems.md

```diff
@@ -17,11 +17,13 @@ For example: `RAS` means that the first dimension (X) points towards the right
 hand side of the head, the second dimension (Y) points towards the Anterior
 aspect of the head, and the third dimension (Z) points towards the top of the
 head.
+The directions are following the *neurological convention*, so the subject's
+left side of the head is also on the left side of the image.
```

Comment:
I think this is meant to say something like:

```suggestion
The directions are considered to be from the subject's perspective.
For example, in the `RAS` coordinate system, a point to the subject's left
will have a negative `x` value.
```

It is possible that others use neurological/radiological to specify this concept, but that's a conflation of coordinates and visualization. Since we don't use it elsewhere, I don't think introducing and defining it adds anything.